web/packages/teleterm/src/ui/appContext.ts

```diff
@@ -212,4 +192,21 @@ export default class AppContext implements IAppContext {
       }
     );
   }
+
+  private subscribeToDeepLinkLaunch() {
+    this.mainProcessClient.subscribeToDeepLinkLaunch(result => {
+      this.deepLinksService.launchDeepLink(result).catch(error => {
+        this.logger.error('Error when launching a deep link', error);
+      });
+    });
+
+    if (process.env.NODE_ENV === 'development') {
+      window['deepLinkLaunch'] = (url: string) => {
+        const result = parseDeepLink(url);
+        this.deepLinksService.launchDeepLink(result).catch(error => {
+          this.logger.error('Error when launching a deep link', error);
+        });
+      };
+    }
```

Comment:
I added this because debugging this with a packaged app is annoying because of how long the feedback loop is. Since it uses `process.env.NODE_ENV === 'development'`, this code won't exist in the packaged app. We do the same thing with `window.teleterm`

https://github.com/gravitational/teleport/blob/b5696c059e82157336c4b4b2f62f22549fbcba1a/web/packages/teleterm/src/ui/appContextProvider.tsx#L32-L36